### DISCUSSION TITLE
How to handle nested scroll container

### DISCUSSION THREAD

**Discussion by enes-get-local** [nolimits4web/swiper#4314] [Q&A]
i m using swiper slides as main layout page. So i want to place components that have own scroller to use it. 

Like this my case 

```
<Swiper> 
   <table> ....... </table>
   // other stuffs
</Swiper>
```
Here , Swiper Slide is main page in the screen and all components wrapped into it.

For example, Table has a lot of data and table body should be scrolling on mobile devices. I can't scroll table body on mobile because swiper slide prevent it. 

How i can handle it ?

**Comment by nolimits4web** [comment]
I would try to wrap such scrollable content within one more Swiper with "Scroll container" setup https://swiperjs.com/demos#scroll-container and also enable `cssMode` on it to keep native scrolling

**Comment by hrvstr** [comment]
I am trying to set this up at the moment. I got it mostly working but only when I set `mousewheel: false` on the parent Swiper and use the cursor drag gesture to scroll. When I set `mousewheel: true` on the parent Swiper scrolling the nested swiper does not work anymore.

I really need to have mouse wheel control on both sadly. Any ideas about what I am doing wrong?

https://codesandbox.io/p/sandbox/swiper-default-forked-rkjnpl?file=%2Findex.html%3A1%2C1

**Comment by hrvstr** [comment]
I am getting closer to a solution. I can disable the mainSwiper mousewheel when I reach the last slide with the nested swiper. However going back up by disabling the mousewheel on the mainSwiper does not work. The "touchStart" or "reachBeginning" event never fires for my nested slider.

```javascript
  mainSwiper.on("reachEnd", () => {
      mainSwiper.mousewheel.disable();
  });

  nestedSwiper.on("touchStart", () => {
    console.log("nested swiper is at the top");
    mainSwiper.mousewheel.enable();
  });
```
